### PR TITLE
Allow custom theme for i3-nagbar

### PR DESCRIPTION
**Pull request by gartens**
It is now possible to set a custom color theme for i3-nagbar.

I tried to keep it as little intrusive as possible. I am looking forward to your feedback.

### CODE REVIEW

File: i3-nagbar/main.c [hunk 1]
```diff
@@ -390,11 +406,20 @@ int main(int argc, char *argv[]) {
                 prompt = i3string_from_utf8(optarg);
                 break;
             case 't':
-                bar_type = (strcasecmp(optarg, "warning") == 0 ? TYPE_WARNING : TYPE_ERROR);
+                if (strcasecmp(optarg, "warning") == 0) {
+                    bar_type = TYPE_WARNING;
+                } else if (strcasecmp(optarg, "custom") == 0) {
+                    bar_type = TYPE_CUSTOM;
+                }
```
Comment (barddes) [inline i3-nagbar/main.c]: Actually its my first time reading through the source code, so I might be wrong. But shouldn't exist some condition making `bar_type = TYPE_ERROR;`?

Comment (gartens) [reply i3-nagbar/main.c]: Actually `bar_type` is by default `TYPE_ERROR` (see line 376). So if the type argument is something we don't know, it is just `TYPE_ERROR`. This is also the current behavior.

**Comment by orestisfl** [comment]
I'll let @Airblader reply to this but I don't think we want this feature. Just letting you know before you put more work into this.

**Comment by Airblader** [comment]
Yeah, I am hesitant to merge this as it pushes i3-nagbar into a direction I'm not sure we want to go. There's plenty of good and customizable tools which are far better than i3-nagbar, which we mostly need for config errors and as an initial configuration for the logout in the default config.

**Comment by gartens** [comment]
Ah I was not aware of those tools. Maybe you can give me some pointers?

**Comment by Airblader** [comment]
@gartens dmenu can be used for such things, or something like zenity etc. I don't know all the tools out there, but i3 nagbar exists mostly for our internal use so we have something to use without requiring dependencies.

Closing this PR as explained above. Thanks for understanding!

**Pull request closed without merging**